src/ui/public/notify/notifier.js

```diff
@@ -39,22 +34,61 @@ function closeNotif(cb, key) {
     // this === notif
     var i = notifs.indexOf(this);
     if (i !== -1) notifs.splice(i, 1);
-    if (this.timerId) this.timerId = clearTO(this.timerId);
+    if (this.timerId) {
+      Notifier.config.clearInterval(this.timerId);
+      this.timerId = undefined;
+    }
     if (typeof cb === 'function') cb(key);
   };
 }
 
+function cancelNotifTimer(cb, key) {
+  return function () {
+    if (this.timerId) {
+      Notifier.config.clearInterval(this.timerId);
+      this.timerId = undefined;
+    }
+
+    if (typeof cb === 'function') cb(key);
```

Comment:
How about a default parameter instead of conditional invocation?

```js
import { noop } from 'lodash';
function wat(cb = noop, key) {
  cb(key);
}
```